Makefile

```diff
@@ -75,20 +75,24 @@ CXXFLAGS += $(ALSA_CFLAGS)
 LDFLAGS += $(ALSA_LDFLAGS)
 endif
 
-ASSISTANT_O = $(AUDIO_SRCS:.cc=.o) ./src/audio_input_file.o ./src/base64_encode.o ./src/json_util.o \
-	./src/run_assistant_audio.o ./src/run_assistant_text.o
-ASSISTANT_AUDIO_O = $(AUDIO_SRCS:.cc=.o) ./src/audio_input_file.o ./src/base64_encode.o ./src/json_util.o \
-	./src/run_assistant_audio.o
-ASSISTANT_FILE_O = $(AUDIO_SRCS:.cc=.o) ./src/audio_input_file.o ./src/base64_encode.o ./src/json_util.o \
-	./src/run_assistant_file.o
-ASSISTANT_TEXT_O = ./src/json_util.o ./src/base64_encode.o ./src/run_assistant_text.o
+CORE_SRCS = ./src/base64_encode.cc ./src/json_util.cc
+
+ASSISTANT_O       = $(CORE_SRCS:.cc=.o) $(AUDIO_SRCS:.cc=.o) \
+                    ./src/audio_input_file.o ./src/run_assistant_audio.o ./src/run_assistant_file.o ./src/run_assistant_text.o
```

Comment:
Should we put this into its own _SRCS?

Reply:
I've changed it so that each source code filename appears only once in the Makefile to ease maintainability.